offer-manager-domain/src/main/groovy/life/qbic/business/offers/create/update/UpdateOffer.groovy

```diff
@@ -115,17 +113,8 @@ class UpdateOffer implements CreateOfferInput, CalculatePrice{
                 maxID = id
             }
         }
+        println maxID
```

Comment:
can be removed

Reply:
can you suggest it so you get a contribution?